web/backend/start.sh

```diff
@@ -0,0 +1,4 @@
+echo " || STARTING SERVER ||"
+
+#Runs Server
+python run.py
```

Comment:
would prefer a more descriptive name for `run.py`

Reply:
sure, i'll name it run_server.py?

Reply:
actually this shell script is probably unnessicary. /etc/hobby-hub/start.sh should handle starting up the server. then whatever additional processes we start will end up managing when to stop the server. 
so you probably should just delete this file all together :p.